int32 can_id
int32 channel
int32 speed
---
int32 response
